src/dso_api/dynamic_api/utils.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+RE_CAMELIZE = re.compile(r"[a-z0-9]_[a-z0-9]")
+
+
+def _underscore_to_camel(match):
+    chars = match.group()  # take complete match, it's only 3 chars
+    return chars[0] + chars[2].upper()
+
+
+def key_to_camel_case(key: str) -> str:
```

Comment:
Misschien `snake_to_camel_case` is beter? Omdat deze functie verwacht alleen snake_case als input?

Reply:
Goede! Fixed!